src/components/Card/Card.scss

```diff
@@ -23,7 +31,18 @@
 }
 
 .header {
-  padding: spacing(loose) spacing(loose) 0;
+  padding: .75rem 1.25rem;
```

Comment:
@HiirenP don't ever use padding or margin in any of the components directly, we want to make all the components generic. In some places we might need less padding/margin and in some more, so its better to not give this values at all. This values will be passed to component using `componentStyle` or `componentClass`